sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala

```diff
@@ -295,10 +295,16 @@ package object dsl {
   object expressions extends ExpressionConversions  // scalastyle:ignore
 
   object plans {  // scalastyle:ignore
-    def table(ref: String): LogicalPlan = UnresolvedRelation(TableIdentifier(ref))
+    def table(parts: String*): LogicalPlan = UnresolvedRelation(parts)
 
-    def table(db: String, ref: String): LogicalPlan =
-      UnresolvedRelation(TableIdentifier(ref, Option(db)))
+    // Cannot use the name `with` since it is reserved
```

Comment:
Since this is the exiting function (moved from a test suite), I guess we don't need this comment.

Reply:
It is now used by 2 test cases, so it will have to be moved out of the test case into test suite. I am ok with that, although this convenience function seem generic enough to go into dsl/package.

Reply:
Yes. It should be the utility function at that test suite. We can bring this to `dsl/package` if needed anytime in the future.